src/main/java/org/firmata4j/fsm/EventPublisher.java

```diff
@@ -0,0 +1,38 @@
+package org.firmata4j.fsm;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author akia
+ * @since 2018-01
+ */
+public class EventPublisher {
```

Comment:
I think we can get by without this class because it makes it impossible to have more than one `FirmataDevice` (`EventListener`) instance per an application and doesn't really do much actual work.